src/entrypoints/options/pages/general/index.tsx

```diff
@@ -1,12 +1,75 @@
+import { toast } from 'sonner'
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog'
+import { Button } from '@/components/ui/button'
+import { ConfigCard } from '../../components/config-card'
 import { PageLayout } from '../../components/page-layout'
 import { ReadConfig } from './read-config'
 import TranslationConfig from './translation-config'
 
+function resetToDefaultConfig() {
+  browser.storage.local.remove('config').then(() => {
+    toast.success('已重置为默认配置，请刷新页面')
+    location.reload()
+  })
+}
+
 export function GeneralPage() {
   return (
-    <PageLayout title={i18n.t('options.general.title')} innerClassName="[&>*]:border-b [&>*:last-child]:border-b-0">
-      <ReadConfig />
```

Comment:
ReadConfig has already been wrapped by ConfigCard I think, you don't need to wrap it again